src-docs/src/views/color_picker/color_palette_picker.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import {
+  euiPaletteColorBlind,
+  euiPaletteForStatus,
+} from '../../../../src/services';
+// @ts-ignore
+import { EuiFormRow } from '../../../../src/components';
+import {
+  EuiColorPalettePicker,
+  EuiColorPalettePickerPaletteProps,
+} from '../../../../src/components/color_picker/color_palette_picker';
+// @ts-ignore
+import { DisplayToggles } from '../form_controls/display_toggles';
+
+const palettesExample1: EuiColorPalettePickerPaletteProps[] = [
+  {
+    value: 'palletteExample1_1',
+    title: 'EUI Color Blind (fixed)',
```

Comment:
We try to use sentence case for everything. Can you rename all these titles so they follow that pattern. For example:
```suggestion
    title: 'EUI color blind (fixed)',
```